### ISSUE TITLE
'0' is not valid for 'Value'. 'Value' should be between 'minimum' and 'maximum'

### ISSUE THREAD

**Issue by Swearie** [Novalys/PokemonGo-Bot-SimpleGUI#171]
I get this error once I choose any location. The program has worked just fine this afternoon (I'm in Italy) and now it's not working anymore. After getting this error, it won't display me nothing but my nickname and level. All other fields are blank. Location is displayed too.
Also, Pokémon are displayed in "My Pokémon List". It won't just start farming.

This is the console log:

> [22:09:57] Adjusting the Settings.
[22:09:57] Trying to Login with Google Token...
[22:10:00] Connected! Server is Ready.
[22:10:00] Attempting to Retrieve Inventory and Player Profile...
[22:10:00] Ready to Work.
[22:10:00] Select Starting Location...
[22:10:35] Starting in Location Lat: 47,62668 Lng: -122,335884
[22:11:57] '0' non è un valore valido per 'Value'. 'Value' deve essere compreso tra 'minimum' e 'maximum'.
Nome parametro: Value

Thank you for this program and for your support!

**Comment by nxy** [comment]
Lat: 47,62668 Lng: -122,335884

They have a comma, you need to change it to a period

**Comment by Swearie** [comment]
>Lat: 47,62668 Lng: -122,335884

Your coordinates have a comma. I believe you need to change it to a period. So it will be..

Lat: 47.62668 Lng: -122.335884

If I choose for example Seattle from the menu it's the same thing, unfortunately...

**Comment by Novalys** [comment]
This is a localization problem since your computer is in French.. Let me see what I can do about this.

**Comment by Swearie** [comment]
> This is a localization problem since your computer is in French.. Let me see what I can do about this.

Thank you! My computer is in Italian and it already was when your program was still working. :)

**Comment by Swearie** [comment]
Update: I experienced the same issue with the v1.2 but moving around on the (Seattle) map, it succesfully worked again as before. I hope this helps you to find exactly what's wrong

**Comment by Dyndrilliac** [comment]
I just encountered this same exception in v1.4, but in a different context. First I encountered issue #201 and then I closed the application and relaunched it. I then proceeded to evolve all my pokemon. Once I got enough XP to level up (I went from 24 to 25) that's when I got this error. I believe it is related to having zero XP. Logs attached below.

[Farming.ErrorLog.20160801065455348.txt](https://github.com/Novalys/PokemonGo-Bot-SimpleGUI/files/393589/Farming.ErrorLog.20160801065455348.txt)
[log.txt](https://github.com/Novalys/PokemonGo-Bot-SimpleGUI/files/393590/log.txt)
